### PR TITLE
ansible: add create-cross-compiler

### PR DESCRIPTION
**Pull request by maclover7**
Converts `setup/cross-compiler` to `ansible`. Will remove
`setup/cross-compiler` upon this landing.

cc @rvagg @nodejs/build @nodejs/jenkins-admins

### CODE REVIEW

**Comment by maclover7** [comment]
@gibfahn Updated with new commit. Yep, I ran the playbook on the machine and everything seems to be ok.

**Comment by rvagg** [comment]
this is for @joaocgreis to review, I'm out of the loop on the cross-compiler

**Comment by rvagg** [comment]
whoops, see https://github.com/nodejs/build/issues/1113#issuecomment-365076072 @maclover7

File: ansible/inventory.yml [hunk 1]
```diff
@@ -78,6 +78,9 @@ hosts:
         win2012r2-x64-2: {ip: 104.209.212.27, port: 61718}
         win2012r2-x64-3: {ip: 104.209.210.218, port: 52109}
 
+    - cloudapp:
+        ubuntu1404-x64-1: {ip: node-msft-cross-compiler-1.cloudapp.net}
```
Comment (joaocgreis) [inline ansible/inventory.yml]: The provider should be `azure_msft`, so something like:

```
- azure:
    msft-ubuntu1404-x64-1: ...
```

The same is true for the Windows servers above, but they were added as a draft by Johan and have not been reviewed yet.

**Comment by maclover7** [comment]
updated @joaocgreis, PTAL -- if okay, will squash and then land as one commit

**Comment by maclover7** [comment]
Landing...

**Comment by maclover7** [comment]
Landed in 5365633

**Pull request closed without merging**